src/smc-webapp/editor.coffee

```diff
@@ -2067,8 +2070,28 @@ class CodeMirrorEditor extends FileEditor
                                 cb(err)
                             else
                                 url = mesg.url + "?nocache=#{Math.random()}"
-                                window.open(url,'_blank')
                                 dialog.find(".salvus-file-printing-link").attr('href', url).text(pdf).show()
+                                if is_subdir
+                                    subdir_texfile = p.head + "/#{base}-sagews2pdf/tmp.tex"
```

Comment:
I tried this code a few times in my dev project, and it failed in every case in various ways.  It never worked.   E.g., when the sagews is in a subdirectory the generated file wasn't even in that subdir.  When the sagews was in the home directory, this subdir_texfile ended up with a / at the front.